files/private-chef-cookbooks/chef-ha-drbd/attributes/default.rb

```diff
@@ -7,6 +7,7 @@
 default['private_chef']['drbd']['shared_secret'] = "promisespromises"
 default['private_chef']['drbd']['device'] = "/dev/drbd0"
 default['private_chef']['drbd']['disk'] = "/dev/opscode/drbd"
+default['private_chef']['drbd']['notify_split_brain_sh'] = '/usr/lib/drbd/notify-split-brain.sh'
```

Comment:
So this appears to change the default location for the script - I'm not sure that's what we want for all users, is it?